model/TriplesManagerService.php

```diff
@@ -175,6 +181,12 @@ protected function isFileReference(Triple $triple)
         $property = $this->getProperty($triple->predicate);
         $range = $property->getRange();
 
+        $uri = $property->getUri();
+
+        if ($uri == MediaService::PROPERTY_LINK) {
```

Comment:
This condition seems dangerous. Returns true means that this value will be handle as a file. But we are not sure if it is a local path, a file ref, a media ref or an tao encoded uri

Reply:
please suggest a different one , this was the only one that worked

Reply:
Maybe if you add also: 

```
if ($uri == MediaService::PROPERTY_LINK && strpos($triple->object, 'file://') !== false) {
```
But I think this logic is really critical and important to be hidden in a `TripplesManagerDoesEverythingService`.

Reply:
does not work with recommendation :(